src/extension/features/accounts/capture-native-search-shortcuts/index.js

```diff
@@ -0,0 +1,93 @@
+import { Feature } from 'toolkit/extension/features/feature';
+import { isCurrentRouteAccountsPage } from 'toolkit/extension/utils/ynab';
+import { isPlatformMac } from 'toolkit/extension/utils/platform';
+
+export class CaptureNativeSearchShortcuts extends Feature {
```

Comment:
I've really wanted to get around to adding a "keyboard shortcuts" feature. Just to future proof this and make it easy to implement, do you mind calling this feature class `KeyboardShortcuts`. Keep all the external naming as you have it, I just want the code and setting to be called KeyboardShortcuts so when I or someone else gets around to adding the future, users using this feature, already have it enabled.